CHANGELOG.md

```diff
@@ -1,16 +1,42 @@
 Changelog
 =========
 
-## 0.6.0 SNAPSHOT (current master)
+## 0.6.0-RC1
+
+### breaking changes
 
 * reorganize maven packages: rename group parent to ohsome-parent (version bump to 2.3), rename local parent to oshdb-parent, and change groupId to org.heigit.ohsome
```

Comment:
do we have a PR number for this change?

Reply:
yeah, #234 and #257 

btw, I see now that _version bump to 2.3_ is also not 100% correct anymore.